src/renderer/containers/Account/AccountOverview/index.tsx

```diff
@@ -5,56 +5,101 @@ import axios from 'axios';
 
 import DetailPanel from '@renderer/components/DetailPanel';
 import Qr from '@renderer/components/Qr';
-import {getActivePrimaryValidatorConfig} from '@renderer/selectors';
+import {useBooleanState} from '@renderer/hooks';
+import {getActivePrimaryValidatorConfig, getManagedAccounts} from '@renderer/selectors';
 import {formatAddress} from '@renderer/utils/address';
 
 import './AccountOverview.scss';
 
 const AccountOverview: FC = () => {
   const {accountNumber} = useParams();
-  const activePrimaryValidator = useSelector(getActivePrimaryValidatorConfig);
   const [balance, setBalance] = useState<number | null>(null);
+  const [loading, setLoading] = useState<boolean>(true);
+  const [signingKeyVisible, toggleSigningKeyVisible] = useBooleanState(false);
+  const activePrimaryValidator = useSelector(getActivePrimaryValidatorConfig);
+  const managedAccounts = useSelector(getManagedAccounts);
+  const managedAccount = managedAccounts[accountNumber];
+
+  useEffect(() => {
+    if (signingKeyVisible) toggleSigningKeyVisible();
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [accountNumber]);
```

Comment:
Added this because when you are on account A and show the signing key, and then you navigate to account B, you may not (most likely probably not) want account B's signing key to be visible.